docs/spark-qualification-tool.md

```diff
@@ -6,12 +6,25 @@ nav_order: 8
 
 # Qualification Tool
 
-The Qualification tool analyzes Spark events generated from CPU based Spark applications to determine 
-if the RAPIDS Accelerator for Apache Spark might be a good fit for GPU acceleration.
+The Qualification tool analyzes Spark events generated from CPU based Spark applications to help quantify
+the expected acceleration of migrating a Spark application to GPU.
+
+The tool first analyzes the operators executed from the CPU log and determines which ones would be qualified to run on
```

Comment:
tool analyzes the CPU event log and determine which operators are likely to run on the GPU.